### DISCUSSION TITLE
Scaling (resizing) of PDF with all the content

### DISCUSSION THREAD

**Discussion by vkovenko** [pymupdf/PyMuPDF#2578] [Q&A] [answered]
Greetings! I wonder if it's possible to resize pdf along with all the content relevant to it (images, text, svg elements, etc.).
Assume that I need to make my pdf 2 times bigger. Any option of how to achieve it using fitz library?

**Comment by JorjMcKie** [comment, marked as answer]
You mean you want to increase the dimension of each page? No problem:
```python
src = fitz.open("input.pdf")
doc = fitz.open()  # new output PDF
for page in src:
    w, h = page.rect.br  # extract input page widht and height
    newpage = doc.new_page(width=2*w, height=2*h)
    newpage.show_pdf_page(newpage.rect, src, page.number)
doc.ez_save("output.pdf")
```
The output will show the original pages stretched by a factor of 2.

**Comment by JorjMcKie** [comment]
Let me transfer this to Discussions.

**Comment by vkovenko** [comment]
Works perfectly, thank you!

**Comment by emarshz** [comment]
@JorjMcKie this goes back to something I posted the other day, but when you resize this, can you not include the annotations with that?
